engines/ic3base.cpp

```diff
@@ -398,16 +399,24 @@ bool IC3Base::block_all()
       reset_solver();
     }
 
-    const ProofGoal &pg = get_next_proof_goal();
+    ProofGoal * pg = get_top_proof_goal();
```

Comment:
are we going to change this later? if not then it should be a pointer to const object

Reply:
Good point! Updated with 9f493ef01f129dbc798555683b27a762a3bf5b98